AlphaWallet/Accounts/ViewModels/AccountsViewModel.swift

```diff
@@ -64,7 +64,6 @@ final class AccountsViewModel {
          walletBalanceService: WalletBalanceService,
          blockiesGenerator: BlockiesGenerator,
          domainResolutionService: DomainResolutionServiceType) {
-
```

Comment:
could u please not remove first space in init method, a bit easier to look at code with it